src/index.html

```diff
@@ -11,7 +11,136 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="form__fieldset">
+        <legend>Personal information</legend>
+        <div class="form__field">
+          <label for="surname">Surname:</label>
```

Comment:
You can wrap input into label as you did for radio buttons, better to use one approach in your solution